humble/distribution.yaml

```diff
@@ -6353,6 +6353,21 @@ repositories:
       url: https://github.com/ros-drivers/transport_drivers.git
       version: main
     status: developed
+  turbojpeg_compressed_image_transport:
+    doc:
+      type: git
+      url: https://github.com/wep21/turbojpeg_compressed_image_transport.git
+      version: main
+    release:
+      tags:
+        release: release/humble/{package}/{version}
+      url: https://github.com/ros2-gbp/turbojpeg_compressed_image_transport-release.git
+      version: 0.1.0-1
+    source:
+      type: git
+      url: https://github.com/wep21/turbojpeg_compressed_image_transport.git
```

Comment:
@wep21 I see mixed Apache Version 2.0 and BSD licensing. What's the background story here? Is this package a subset of an older ROS 1 package?

Reply:
@hidmic I used some part of code from compressed image transport whose license is [BSD](https://github.com/ros-perception/image_transport_plugins/blob/f0f40fbcee376160106f39aee3144736445c21d9/compressed_image_transport/package.xml#L9) Is it better to unify license definition?

Reply:
@wep21 I see. If you are using BSD-3 licensed code in an Apache 2.0 licensed package then yes, I think we should include both LICENSES and mention them in the `package.xml`. 

You could also stick to BSD-3. Your call.